docset/winserver2019-ps/failoverclusters/New-Cluster.md

```diff
@@ -168,6 +168,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ManagementPointNetworkType
+Specifies the network configuration used to determine IP address settings.
+
+The acceptable values for this parameter are:
+
+```yaml
```

Comment:
I would suggest against using a yaml block to represent the options. I would follow something more like the `AdministrativeAccessPoint` template and use  a bulleted list.

Reply:
I also agree with removing the yaml block.